test/fuzzTesting/createRandomTest.cpp

```diff
@@ -139,6 +141,14 @@ int main(int argc, char *argv[])
 			else
 				fillRandomTest(dev::test::doVMTests, (filltest) ? testFillString : c_testExampleVMTest, filldebug);
 		}
+		else
+		if (testSuite == "RLPTests")
+		{
+			if (checktest)
+				return checkRandomTest(dev::test::doStateTests, testmValue, debug);
```

Comment:
flag:  in progress
I will update this PR when https://github.com/ethereum/cpp-ethereum/pull/2375 will be merged